app/assets/stylesheets/application.css.scss.erb

```diff
@@ -120,7 +120,7 @@ td, th {
 }
 .our-search{
     margin: 0 !important;
-    background-color: rgba(0, 0, 0, 0.64);
+    background-color: rgba(0, 0, 0, 0.9);
 }
 .before-scroll{
```

Comment:
I think it will be nice to have each of the CSS classes spaced at least a line from each other.